lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -327,12 +327,7 @@ function NearbyView({
           />
         </InvisibleA11yLabel>
       )}
-      <NearbySidebarContainer
-        className="base-color-bg"
-        style={{ marginBottom: 0 }}
-      >
-        {/* This is used to scroll to top */}
-        <div aria-hidden ref={firstItemRef} />
+      <div style={{ padding: '1em' }}>
```

Comment:
This now means the location field doesn't scroll with the rest of the items. This is causing problems in configs that hide the location field but also is a behavior that I've seen explicitly asked _not _ to have

Reply:
It's also causing this issue when jumping to a specific stop id <img width="484" alt="Screenshot 2025-05-28 at 2 45 54 PM" src="https://github.com/user-attachments/assets/ac5ffee7-f0f1-4702-9830-f29fb7d17206" />